nautobot_design_builder/design.py

```diff
@@ -550,7 +587,16 @@ def __init__(
 
         self.journal = Journal(design_journal=journal)
 
-    def get_extension(self, ext_type, tag):
+    def decommission_object(self, object_id, object_name):
+        """This method decommissions an specific object_id from the design instance."""
+        self.journal.design_journal.design_instance.decommission(
+            local_logger=get_logger(__name__, self.job_result), object_id=object_id
```

Comment:
I believe this may be causing the duplicate log lines... `get_logger` always returns the same logger (base on name), but also always adds an additional handler for the `JobResult`. Each time `get_logger` is called with the same name it will add a new handler that will log to the `JobResult`.

It would be better to add a logger to the base `Builder` and then jus re-use that.

Reply:
you were right